queries/python.scm

```diff
@@ -80,6 +80,14 @@
   (_) @value
 ) @_.domain
 
+;;!! for aaa in bbb:
+;;!             ^^^
+;;!  ---------------
+(for_statement
+  right: (identifier) @value
```

Comment:
why only identifier?

Reply:
oh nice catch. I guess that's the danger of the new "parse tree" action: we have a new vector for copy-paste errors 😅